beeai/common/models.py

```diff
@@ -163,6 +163,53 @@ class TriageOutputSchema(BaseModel):
         description="Associated data"
     )
 
+    def format_for_comment(self) -> str:
+        """Format the triage result in a human-readable format for Jira comments."""
+        resolution = f"*Resolution*: {self.resolution.value}\n"
+
+        match self.data:
+            case BackportData():
```

Comment:
Is this an [undocumented](https://docs.pydantic.dev/latest/concepts/models/#structural-pattern-matching) feature? No arguments match the same model with arbitrary attribute values?